documentation/secure_storage.md

```diff
@@ -235,24 +225,14 @@ or crypto unit according to the method defined by your SoC vendor.
 
 ## Future Work
 
-- **TEE file renaming attack detection**
-
-OP-TEE creates a specific folder under the TA's folder for each TEE file in
-Linux file system and use the filename of the TEE file as the folder's name.
-If an attacker directly rename the name of a TEE file folder, the renamed
-TEE file is still a valid TEE file in OP-TEE.
-
-A solution to detect the attack is using TEE filename as AAD when calculating
-the tag of meta file.
-
 - **Rollback attack detection**
 
-An attacker can backup each version of a TEE file directly from Linux file
-system and can replace the TEE file by an old version one sooner or later.
+An attacker can backup the whole `/data/tee` folder and restore it at later
+time.
 
-The basic idea of detecting rollback attack is to add write counter both in
-meta file and another storage which has anti-rollback capability such as eMMC
-RPMB partition.
+The basic idea of detecting rollback attack is to store a hash representing the
```

Comment:
Could be worth mentioning that there's also a counter that can be used for this purpose.